CHANGELOG.md

```diff
@@ -15,15 +15,22 @@ Please visit [cucumber/CONTRIBUTING.md](https://github.com/cucumber/cucumber/blo
 ### Added
 
 ### Changed
-* Going forward the minimum ruby version for all cucumber based gems is 2.3
-([luke-hill](https://github.com/luke-hill))
+* Going forward the minimum ruby version for all cucumber based gems is 2.4
```

Comment:
Is this strictly true for all cucumber gems (Are we going to make the minimum supported ver 2.4?

I don't mind either way, but I didn't feel comfortable making that decision as I know a reasonable amount of people use 2.3 and I'm still new-ish here.

I'm definitely all for it, as it makes support 10* easier. But just wanted to confirm this was for *all* cucumber gems, not just the main one.

Reply:
I reverted to 2.3